compiler/src/main/java/com/squareup/anvil/compiler/codegen/CodeGenerationExtension.kt

```diff
@@ -79,11 +79,11 @@ internal class CodeGenerationExtension(
 
     // This restarts the analysis phase and will include our files.
     return RetryWithAdditionalRoots(
-      bindingTrace.bindingContext,
-      module,
-      emptyList(),
-      listOf(codeGenDir),
-      true
+      bindingContext = bindingTrace.bindingContext,
+      moduleDescriptor = module,
+      additionalJavaRoots = emptyList(),
+      additionalKotlinRoots = listOf(codeGenDir),
+      addToEnvironment = true
```

Comment:
They've added another parameter with a default argument in between instead of at the end. Therefore I had to use named arguments.